dev-client/src/components/projects/CreateProjectView/Form.tsx

```diff
@@ -1,9 +1,8 @@
-import {HStack, Input, Text, VStack} from 'native-base';
+import {Fab, HStack, Input, Text, VStack} from 'native-base';
```

Comment:
note: the `SaveFAB` was mainly used for styling, i have done this styling in our theme instead of a component so we can directly use the `Fab` component. if we have need for more than one FAB style we can figure out how to use variants.